Mathlib/RingTheory/MvPolynomial/Homogeneous.lean

```diff
@@ -261,18 +261,22 @@ theorem sub (hφ : IsHomogeneous φ n) (hψ : IsHomogeneous ψ n) : IsHomogeneou
 
 end CommRing
 
+/-- The homogeneous degree bounds the total degree.
+
+See also `MvPolynomial.IsHomogeneous.totalDegree` when `φ` is non-zero. -/
+lemma totalDegree_le_of_isHomogeneous (hφ : IsHomogeneous φ n) : φ.totalDegree ≤ n := by
```

Comment:
Whoops, I didn't notice that you also proposed a namechange. But we're in the `IsHomogeneous` namespace. So I think we should go with the short name that allows for dot-notation. Do you agree?
```suggestion
lemma totalDegree_le (hφ : IsHomogeneous φ n) : φ.totalDegree ≤ n := by
```

Reply:
Yes, I prefer your name!